lib/async_hooks.js

```diff
@@ -49,6 +49,9 @@ const init_symbol = Symbol('init');
 const before_symbol = Symbol('before');
 const after_symbol = Symbol('after');
 const destroy_symbol = Symbol('destroy');
+const emitBeforeN = emitHookFactory(before_symbol);
```

Comment:
Only the `emitDestroyN` hook is used only once, the other two are used in more than one spot. I made them all const for consistency.